src/components/html/EcAdminSettingsForm.html

```diff
@@ -77,6 +77,21 @@
             </div>
           </template>
 
+          <component
+            v-for="({
+              field,
+              name,
+              schema,
+              component,
+              data
+            }) in parseAdminSettingsField(config)"
+            v-if="component && checkComplexSchema(schema) && field === config.field"
+            v-bind="{ name, schema }"
+            v-model="data[field]"
+            :key="name"
+            :is="component"
+          />
```

Comment:
Tá repitindo muito código, não seria melhor subtituir o `<a-form-item>` por um `<component :is="checkComplexSchema(schema) && field === config.field ? 'div' : 'a-form-item'">`?